src/app/components/accounts/consent-form/consent-form.component.ts

```diff
@@ -12,21 +13,24 @@ import {MESSAGE_TYPES} from '@app/_models';
 })
 export class ConsentFormComponent implements OnInit {
     FormData: FormGroup;
+    currentDate: string;
 
     constructor(private router: Router,
                 private route: ActivatedRoute,
                 private builder: FormBuilder,
                 private consentService: ConsentService,
-                private messageService: MessageService) {
+                private messageService: MessageService,
+                private datePipe: DatePipe) {
     }
 
     ngOnInit(): void {
+        this.currentDate = this.datePipe.transform(new Date(), 'MM/dd/yyyy');
```

Comment:
This page is not important enough to add a provider to the entire project because of it.
Just use `(new Date()).toDateString()`. The format doesn't really matter and it might even be better to have a more human-readable format.
Also, remove the local variable and directly put this in the form control.